src/loaders/ObjectLoader.js

```diff
@@ -466,8 +466,21 @@ THREE.ObjectLoader.prototype = {
 
 				case 'PerspectiveCamera':
 
-					object = Object.assign(
-							new THREE.PerspectiveCamera(), data );
+					object = new THREE.PerspectiveCamera(
+							data.fov, data.aspect, data.near, data.far );
+
+					if ( data.focus !== undefined ) object.focus = data.focus;
+					if ( data.zoom !== undefined ) object.zoom = data.zoom;
+
+					if ( data.filmGauge !== undefined ) {
+
+						if ( data.view !== null )
+							object.view = Object.assign( {}, data.view );
```

Comment:
I'm doing a quick PR to decouple them.  Just a sec.

BTW I've had tons of experience with file converters, the less assumjptions made than what is directly in front of the reader the better for reliability.